Small formatting changes, update of version, update of copyright year (#1794)
modelica/fmi-standard: docs/6___appendix.adoc

For contributors please refer to the issue tracking system https://github.com/modelica/fmi-standard/milestone/18

> #### A.3.7	FMI 2.0.4 maintenane release: changes and contributors

Reviewer: ```suggestion
#### A.3.7	FMI 2.0.4 maintenance release: changes and contributors
```

> The changes w.r.t. FMI 2.0.2(clarifications and bugfixes) are summarized on the release page for FMI 2.0.4 https://github.com/modelica/fmi-standard/releases/tag/v2.0.4 

Reviewer: ```suggestion
The changes w.r.t. FMI 2.0.2 (clarifications and bugfixes) are summarized on the release page for FMI 2.0.4 https://github.com/modelica/fmi-standard/releases/tag/v2.0.4 
```

For contributors please refer to the issue tracking system https://github.com/modelica/fmi-standard/milestone/22

> For the Backported “Terminals and Icons” and “Build Configuration” Features refere to the contributors list to FMI3.0 https://fmi-standard.org/docs/3.0/#_contributors 

Reviewer: ```suggestion
For the backported “Terminals and Icons” and “Build Configuration” features refer to the contributors list of FMI 3.0 https://fmi-standard.org/docs/3.0/#_contributors 
```
